pynpoint/util/multipca.py

```diff
@@ -145,20 +161,34 @@ def run_job(self,
             Output residuals.
         """
 
-        residuals, res_rot = pca_psf_subtraction(images=self.m_star_reshape,
-                                                 angles=self.m_angles,
-                                                 pca_number=int(tmp_task.m_input_data),
-                                                 pca_sklearn=self.m_pca_model,
-                                                 im_shape=self.m_im_shape,
-                                                 indices=self.m_indices)
+        if isinstance(tmp_task.m_input_data, tuple):
```

Comment:
Add a comment to explain this if condition

Reply:
Added the comment.